Dockerfile

```diff
@@ -20,7 +20,9 @@ RUN dpkg --add-architecture i386 && \
     apt-get install -y libncurses5:i386 libc6:i386 libstdc++6:i386 lib32gcc1 lib32ncurses5 lib32z1 zlib1g:i386 && \
     apt-get install -y --no-install-recommends openjdk-8-jdk && \
     apt-get install -y git wget zip && \
-    apt-get install -y qt5-default
+    apt-get install -y ruby-full build-essential dh-autoreconf
+
+RUN gem install fastlane -NV --no-document
```

Comment:
Nice work getting fastlane into linux!

Reply:
Yeah, it should be easier in the future: they're supposed to be working on a multi-platform support or at least, they've got an issue open for it! (https://github.com/fastlane/fastlane/issues/11687)

For reference, I found the fix here:
https://github.com/fastlane/fastlane/issues/11579